bundle-utils.lisp

```diff
@@ -30,6 +30,9 @@
           ("BUNDLE:SUPPORT;USER;**;*.*.*" ,(application-support-directory :user))
           ("BUNDLE:SUPPORT;LOCAL;**;*.*.*" ,(application-support-directory :local)))))
 
+;; FIXME: Compile time error with CCL setting up logical-pathname-translations
+;; Bundle
+#+(or)
```

Comment:
I'd prefer a patch that fixes this functionality. Or, perhaps, splits bundle-utils into a separate system.

Reply:
This evening I will investigate a solution that fixes this functionality. I'll pull the latest patches from `master` precede from there. Thank you for your feedback.